frontend/src/components/Footer.test.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+  This file is part of Edgehog.
+
+  Copyright 2023 SECO Mind Srl
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  SPDX-License-Identifier: Apache-2.0
+*/
+
+import { it, expect } from "vitest";
+import { screen } from "@testing-library/react";
+import { renderWithProviders } from "setupTests";
+
+import Footer from "./Footer";
+
+it("displays correct app name and app version", () => {
+  renderWithProviders(
+    <Footer
+      appName="appName"
+      appVersion="appVersion"
+      homepageUrl="https://edgehog.io/"
+      repoUrl="https://github.com/edgehog-device-manager/edgehog"
+      issueTrackerUrl="https://github.com/edgehog-device-manager/edgehog/issues"
+    />,
+  );
+
+  const appNameElement = screen.getByText("appName");
+  const appVersionElement = screen.getByText("appName");
+
+  expect(appNameElement).toBeInTheDocument();
```

Comment:
We can use [`toBeVisible`](https://github.com/testing-library/jest-dom#tobevisible) to check if an element is currently visible to the user.
```suggestion
  expect(appNameElement).toBeVisible();
```